src/main/java/model/domain/Player.java

```diff
@@ -0,0 +1,42 @@
+package model.domain;
+
+import model.VO.Name;
+import model.VO.Position;
+import model.VO.Result;
+
+public class Player {
+    private final Name name;
+    private Result result;
+    private Position position;
+
+    public Player(final Name name, final Position position) {
+        this.name = name;
+        this.position = position;
+    }
+
+    public Name getName() {
+        return name;
+    }
+
+    public Result getResult() {
+        return result;
+    }
+
+    public void saveResult(final Result result) {
+        this.result = result;
+    }
+
+    public boolean isPlayerName(Name other) {
+        return other.isSame(this.name);
+    }
```

Comment:
```suggestion
    public boolean isPlayerName(Name other) {
        return this.name.isSame(other);
    }
```

위와 같이 짠다면 NPE를 방어할 수 있습니다. 외부에서 들어오는 값은 무엇이 될지 확신하기 어려우니까요 😉

Reply:
앗 그렇군요... 순서가 상관없다고 생각해 개인적으로 더 보기 좋은 방법으로 작성했는데 NPE를 이런 식으로 방어할 수 있을 줄은 몰랐습니다. 수정하도록 하겠습니다!